changelog/243.breaking.rst

```diff
@@ -0,0 +1,29 @@
+The function ``set_attenuation_coefficients`` was renamed to ``_set_attenuation_coefficients`` to indicate that it is a helper function and not intended to be part of the public API in ``radial.py``.
```

Comment:
The goal of the changelog is to tell users what has been broken and how they can fix their code.